psi4/src/export_mints.cc

```diff
@@ -316,10 +333,8 @@ void export_mints(py::module& m)
         .value("Bohr", Molecule::Bohr)
         .export_values();
 
-    typedef void (Matrix::*matrix_multiply)(bool, bool, double, const SharedMatrix&,
-                                            const SharedMatrix&, double);
-    typedef void (Matrix::*matrix_diagonalize)(SharedMatrix&, std::shared_ptr<Vector>&,
-                                               diagonalize_order);
+    typedef void (Matrix::*matrix_multiply)(bool, bool, double, const SharedMatrix&, const SharedMatrix&, double);
+    typedef void (Matrix::*matrix_diagonalize)(SharedMatrix&, std::shared_ptr<Vector>&, diagonalize_order);
```

Comment:
I feel like `export_mints` keeps getting reformatted in different ways. I think this happened with @ashutoshvt's PR, but we need to make sure were keeping to our clang-format parameter set.

Reply:
The Neovim autoformat function uses `.clang-format` in the root directory.